Documentation/admin/upgrade.md

```diff
@@ -15,10 +15,30 @@ Please note that the update payload process may affect any or all components in
 
 To learn more about how this process is executed, read about [Operators][operators].
 
+## Manually update Container Linux
+
+To provide the most secure operating system, Tectonic automatically updates Container Linux. To manage the update process for Container Linux, rather than allowing it to run automatically, configure the system to allow manual updates.
+
+First, scale the container-linux-update-operator deployment to 0 to pause automated updates.
+* Go to *Workloads > Deployments*, and select the container-linux-update-operator.
+* Set the desired count to 0 Pods, and click *Save*.
+
+Then, mask update-engine to prevent it from downloading or installing updates.
+For more information, see [Disable Automatic Updates Daemon][disable-auto] in the Container Linux reboot strategies guide.
+
+Reset the deployment.
```

Comment:
is this telling you to set it back to 0 again? or are you saying scale it back up? back up to what?